### PR TITLE
Problem: It's not clear non-live and byzantine slashing are independent

### PR DESCRIPTION
**Pull request by yihuang**
Solution:
- Clarify a bit

### CODE REVIEW

File: docs/getting-started/punishments.md [hunk 1]
```diff
@@ -87,16 +87,12 @@ When a jailed validator wishes to resume normal operations (after `account.jaile
...
 
-### Slashing
+### Byzantine Slashing
 
-Validators are responsible for signing or proposing block at each consensus round. It is important that they maintain
-excellent availability and network connectivity to perform these tasks. A penalty performed by the slashing module
-should be imposed on validators' misbehavior reinforce this. Similar to jailing, a validator is slashed if they make a
-byzantine fault. If a validator makes multiple faults in `UNBONDING_PERIOD`, they'll only be slashed once for the worst
-fault in that period. The funds to be deducted are calculated based on `BYZANTINE_SLASH_PERCENT`.
+Validators are responsible for signing or proposing block at each consensus round. A penalty should be imposed on validators' misbehavior to reinforce this. When there is byzantine fault detected, they are immediately slashed other than jailed. During the jailing time, it won't be slashed again for other byzantine faults. The funds to be deducted are calculated based on `BYZANTINE_SLASH_PERCENT`.
```
Comment (devashishdxt) [inline docs/getting-started/punishments.md:92]: Why did you remove the newlines. They're there because its easy to read on IDE if we use newlines.

File: docs/getting-started/punishments.md [hunk 2]
```diff
@@ -51,15 +51,15 @@ byzantine fault, that evidence should be verified. Also, it should be checked th
...
-### Inactivity Punishment
+### Inactivity Slashing
+
+It is important that the validators maintain excellent availability and network connectivity to perform their tasks. A penalty should be imposed on validators' misbehavior to reinforce this.
 
 When a validator fails to successfully sign `MISSED_BLOCK_THRESHOLD` blocks in last `BLOCK_SIGNING_WINDOW` blocks, it is
-immediately punished by deducting funds from their bonded and unbonded amount and removing them from active validator
-set if the resulting bonded amount is still greater than minimum required stake. The funds to be deducted are calculated
-based on `LIVENESS_SLASH_PERCENT`.
+immediately punished by deducting funds from their bonded and unbonded amount and removing them from active validator set. The funds to be deducted are calculated based on `LIVENESS_SLASH_PERCENT`.
```
Comment (devashishdxt) [inline docs/getting-started/punishments.md:59]: Don't remove newlines.

**Review by devashishdxt — COMMENTED** [review]
Looks good. Just a few comment to add newlines back.

**Review by tomtau — APPROVED** [review]
looks ok. for newlines, this gets rendered anyway; there are probably other files with a bit inconsistent spacing -- perhaps open an issue for guidelines or if there's some automated markdown style check/formatter
